src/general/requiredlabels/constraint.tmpl

```diff
@@ -38,5 +38,8 @@ spec:
                       the regular expression.
   targets:
     - target: admission.k8s.gatekeeper.sh
-      rego: |
-{{ file.Read "src/general/requiredlabels/src.rego" | strings.Indent 8 | strings.TrimSuffix "\n" }}
+      code:
+      - engine: Rego
```

Comment:
During the community call, we said let's keep the cel code for this policy and remove the changes for k8spspallowprivilegeescalationcontainer since that one has issues with the exemptImages, this policy doesn't.

Without the cel code, we cant validate this PR actually works.

you can use the code https://github.com/open-policy-agent/gatekeeper-library/pull/519#discussion_r1594342487 and update the second expression using the same logic.

Reply:
Appologies for the misunderstanding. I thought the decision was to riase a different PR for policy. Added the policy here.